internal/jimm/ssh/ssh.go

```diff
@@ -0,0 +1,48 @@
+// Copyright 2025 Canonical.
+
+package ssh
+
+import (
+	"context"
+
+	"github.com/canonical/jimm/v3/internal/dbmodel"
+	"github.com/canonical/jimm/v3/internal/errors"
+	"github.com/canonical/jimm/v3/internal/openfga"
+)
+
+type identityManager interface {
```

Comment:
We should export these types to indicate (at least to ourselves) that these interfaces form part of the API of our package and external packages rely on their definition.